i18n/zh/docusaurus-plugin-content-docs/current/troubleshooting/troubleshooting.md

```diff
@@ -19,3 +19,18 @@ title: 故障定位
   ```shell
```

Comment:
Hi @liangyongzhenya, would you like to translate the first case too? Great thanks:)

Reply:
Okay, it's been translated!

Reply:
BTW, please merge the commits and sign the DCO.